src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -50,20 +50,29 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
             }
         }
 
-        protected virtual string GetKey(TypeInfo typeInfo)
+        protected virtual string GetResourcePrefix(TypeInfo typeInfo)
         {
-            return GetKey(typeInfo, _hostingEnvironment.ApplicationName, _resourcesRelativePath);
+            return GetResourcePrefix(typeInfo, _hostingEnvironment.ApplicationName, _resourcesRelativePath);
         }
 
-        protected virtual string GetKey(TypeInfo typeInfo, string location, string resourcesRelativePath)
+        /// <summary>
+        /// Constructs the baseName of a resource for use in retrieving it.
+        /// </summary>
+        /// <param name="typeInfo"></param>
+        /// <param name="location"></param>
+        /// <param name="resourcesRelativePath"></param>
+        /// <remarks>If we have a resourcePath we should return "AppName.Resourcepath.(TypeName.trim())"
```

Comment:
Oh duh of course it's product code 😄 I was jumping too much between two files...